### ISSUE TITLE
Black screen(s) when playing Borderlands 2

### ISSUE THREAD

**Issue by Dibler94** [lucasassislar/nucleuscoop#47]
I've tried using Alpha 5 and 6 to play Borderlands 2, and get audio with no visuals. Is there some setting I need to change that I'm not aware of? The game runs fine without Nucleus.

**Comment by Dibler94** [comment]
So when I try extending the screen I end up with the one on my laptop working while the one on the TV I have it plugged into stays black. This is the case no matter which player is assigned to which screen.

**Comment by ryantimjohn** [comment]
I have the same issue. I'm wondering if we both might be using under powered systems? You used the word laptop, I'm also using a portable setup.

**Comment by tbare598** [comment]
Can you run the game normally, using the TV as the display?

**Comment by ryantimjohn** [comment]
Yes, the game works just fine outside of Nucleus Coop. I looked at resources and the CPU resource use goes up to 100% when both games start. I have an Intel Core i5-5200U CPU at 2x 2.20Ghz with a NVidia GeForce 940A (1Gb, rare and kind of weird card.)

**Comment by tbare598** [comment]
Well poor performance shouldn't cause a screen to be completely black. Though game might run sluggish.

Are you able to use NecleusCoop to open the game for 2 players in split screen? By that, I mean 2 players on the same screen.

**Comment by ryantimjohn** [comment]
Right you are. To test this, I ran the game twice by creating two shortcuts, changing properties to include a always focus, a controller offset and a new savegameID. It worked just fine. So, you're right, this isn't graphics card issue.

**Comment by tbare598** [comment]
So there is an easier way to check if you can run 2 players on the same screen(your laptop screen, preferably), and test the necleuscoop tool. After you select the number of players, you'll see in the UI the screens that you can place your players on. From there, click on this image(it's kinda hard to see because it is missing the background): https://github.com/lucasassislar/nucleuscoop/blob/master/Master/NucleusGaming/Resources/fullscreen.png 
After clicking that, it will change the split screen mode selected, so that you can place the 2 players on the same screen. After you place the players on the same screen hit PLAY. 


Wait a second. I just remembered a bug that I fixed, in my own fork of this repo, where the video config file wasn't getting set properly. I'm not at my computer right now, so I can't confirm the location, or the exact name of the file, but I think it is here:
*Drive*:\Users\*username*\My Documents\My Games\Borderlands 2\WillowGame\Config\WillowEngine.ini

And you want to set at least these values:
WindowedFullscreen  = true (or 1)
Fullscreen = false (or 0)
ResX = (your horizontal pixel width) 
ResY = (your vertical pixel heigth) 
bPauseOnLossOfFocus = false (or 0)
bPauseLostFocusWindowed = false (or 0)
bMuteAudioWhenNotInFocus = false (or 0)
bPauseOnLossOfFocus = false (or 0)
bMuteAudioWhenNotInFocus = false (or 0)

**Comment by ryantimjohn** [comment]
Thanks so much, will check that out.

**Comment by tbare598** [comment]
Okay, so I looked into it, and I don't believe the config file was the problem, not for borderlands at least. It was a problem with Left4Dead 2, which I believe the maintainer of this project recently addressed in his latest commit.

I'll have to think about this a bit more. I've always had problems connecting a PC to TVs to play games, since they often don't provide the same support for video protocols as computer monitors do. Especially with weird video resolutions... Wait a second. NecleusCoop has a lot of problems with different resolutions. What resolution does your monitor and TV support? Try setting the computer monitor to the same resolution as the TV, before starting the game.

**Comment by ryantimjohn** [comment]
Hi! So, just to specify: I am not the OP. I am not hooking the game up to a TV, just running off the integrated monitor in my all in one.

**Comment by tbare598** [comment]
So from what you are saying,  you only using one screen? Would you mind describing your situation in a little more detail?

**Comment by ryantimjohn** [comment]
So, I am running an all-in-one with a 1920x1080 display. Integrated Intel
graphics and a dedicated Nvidia 940A card. (Lenovo Yoga Home 900) I have
set both "Instances" of Borderlands2 under NucleusCoop/Data to run on the
Nvidia card.
When I load Nucleus Coop, I click on 2, then the horizontal split, then
press play. Both instances of Borderlands 2 boot up and properly resize to
the top and bottom halves of the screen. But both screens are completely
black. Strangely enough, sound is preserved. And I can hear it load into
the menu, so have started a new game and heard the opening credits. So,
everything's running, it's just displaying black.
